### PR TITLE
:warning: add tumbleweed flavor

### LINKED ISSUE #65: 🐧 tumbleweed flavor
Currently the image definition is there, but lacks few things in order to boot properly. This card is to make the tumbleweed flavor functional and into releases.

- opensuse should then track `tumbleweed`
- new flavor "opensuse-leap" to track opensuse leap
- update docs, this is a breaking change in our repos (opensuse tracks tumbleweed now, and redirect users of leap to the new images)

### PR DESCRIPTION
**Pull request by Itxaka**
**What this PR does / why we need it**:

 - Adds new opensuse-tumbleweed and opensuse-tumbleweed-arm-rpi flavors
 - Renames opensuse to opensuse-leap
 - Adapt earthfile to new names
 - Adapt CI to new names

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #65 

 - [x] docs for core images
 - [x] docs for provider-kairos images (needs another PR under provider-kairos to change naming in there)
 - [x] copy artifacts from core-opensuse into core-opensuse-leap
 - [x] copy signatures from core-opensuse into core-opensuse-leap
 - [x] copy artifacts from core-opensuse-arm-rpi into core-opensuse-leap-arm-rpi
 - [x] copy signatures from core-opensuse-arm-rpi into core-opensuse-leap-arm-rpi

### CODE REVIEW

**Comment by Itxaka** [comment]
If merged, this requires changing the required tests from `opensuse` to `opensuse-leap`

File: .github/workflows/image-arm.yaml
Comment (mudler) [inline .github/workflows/image-arm.yaml:24]: leftover? same flavor name here

Comment (Itxaka) [reply .github/workflows/image-arm.yaml:24]: arrrrgh

File: images/Dockerfile.opensuse-tumbleweed
Comment (mudler) [inline images/Dockerfile.opensuse-tumbleweed:4]: we can drop this, it is a leftover - not really used

Comment (mudler) [reply images/Dockerfile.opensuse-tumbleweed:4]: I know I know.. not part of these changes =) .. buuut

**Review by mudler — APPROVED** [review]
Looks good. just as a reminder, we have to create image repos too

File: images/Dockerfile.opensuse-tumbleweed
Comment (Itxaka) [reply images/Dockerfile.opensuse-tumbleweed:4]: yeah I was wondering why this were being used.....will drop them

**Comment by Itxaka** [comment]
> Looks good. just as a reminder, we have to create image repos too

I have no perms in the quay org so.... :D

**Comment by Itxaka** [comment]
updated docs to move references to core-opensuse to core-opensuse-leap and added core-opensuse-tumbleweed to version matrix

**Comment by mudler** [comment]
Marking with :warning:, it's a major change as needs in the release note a special mention to redirect `opensuse` users to `opensuse-leap`

**Review by mudler — DISMISSED** [review]
Looks good here!

**Comment by Itxaka** [comment]
core-opensuse artifacts copied into core-opensuse-leap artifacts

**Comment by Itxaka** [comment]
ok this is good for now, once this is merged a new tumbleweed image will be pushed and then I can retest https://github.com/kairos-io/provider-kairos/pull/193

File: docs/content/en/docs/Installation/raspberry.md
Comment (mudler) [inline docs/content/en/docs/Installation/raspberry.md:29]: need to change also the version too?

**Review by mudler — APPROVED** [review]
Looking good!

**Comment by Itxaka** [comment]
Omg, can't we set a mirror for the downloads? Terrible failures all the time due to download issues with suse repos

**Comment by mudler** [comment]
yeap, we need to optimize some targets, definitely there is room for improvement #265

**Comment by Itxaka** [comment]
unfortunately the job [upgrade-latest-with-cli-test (opensuse-leap)](https://github.com/kairos-io/kairos/actions/runs/3997276547/jobs/6866881832#logs) wont pass until we release a new version as that uses the latest release but tries to find the iso by the new name. Im wary of changing that and having to change that back again.

cc @kairos-io/maintainers thougths? Should I change that back to opensuse and then do another PR to change it to opensuse-leap once we release 1.5.0?

**Pull request merged**
